Engine/src/Engine/SceneBuilder/Scene.h

```diff
@@ -99,14 +110,37 @@ namespace Engine
         int m_windowWidth{1920};
         int m_windowHeight{1080};
 
+        //Added this to keep track of the entities created
+        std::unordered_map<std::string, entt::entity> m_entityHandles;
```

Comment:
the registry already keeps track of entities created, and entities have an associated TagComponent

Reply:
fixed